src/services/activityReports.js

```diff
@@ -299,30 +299,18 @@ export function activityReportByLegacyId(legacyId) {
   });
 }
 
-export async function populateRecipientInfo(activityRecipients, grantPrograms) {
-  /*
-      Hopefully this code is temporary until we figure
-      out why joining programs to grants causes issues.
-  */
+export function populateRecipientInfo(activityRecipients) {
   return activityRecipients.map((recipient) => {
-    if (recipient.grant && grantPrograms.length) {
-      const programsToAssign = grantPrograms.filter((p) => p.grantId === recipient.grantId);
-      // Programs.
-      Object.defineProperty(
-        recipient.grant,
-        'programs',
-        {
-          value: programsToAssign,
-          enumerable: true,
-        },
-      );
+    if (recipient.grant) {
+      const programsToAssign = (recipient.grant.programs || []).map((p) => p.programType);
 
       // Program Types.
       const programTypes = programsToAssign && programsToAssign.length > 0
         ? [...new Set(
-          programsToAssign.filter((p) => (p.programType))
-            .map((p) => (p.programType)).sort(),
+          programsToAssign.filter((p) => (p))
+            .map((p) => (p)).sort(),
```

Comment:
does this do anything: `.map((p) => (p))`, when you have already filtered it?

Reply:
I think it checks for a value and sorts, not sure why I had it there before